common/src/main/java/com/wynntils/models/lootrun/LootrunModel.java

```diff
@@ -509,11 +574,37 @@ public void addToRedBeaconTaskCount(int changeAmount) {
         redBeaconTaskCountStorage.touched();
     }
 
-    public void resetRedBeaconTaskCount() {
+    private void resetRedBeaconTaskCount() {
         redBeaconTaskCountStorage.get().remove(Models.Character.getId());
         redBeaconTaskCountStorage.touched();
     }
 
+    private void resetMissions() {
+        List<String> missions = missionStorage.get().get(Models.Character.getId());
+        if (missions == null) return;
+
+        missions.clear();
+        missionStorage.get().put(Models.Character.getId(), missions);
+        missionStorage.touched();
```

Comment:
```suggestion
        List<String> missions = missionStorage.get().getOrDefault(Models.Character.getId(), new ArrayList<>());
        missions.clear();
        missionStorage.touched();
```

The put is not needed, it's already a reference from the storage itself. The other change is just a nit.